packages/utilities/fast-gestures/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    extends: ["@microsoft/eslint-config-fast-dna", "prettier"],
```

Comment:
We dropped the `-dna` from our branding so probably could do the same here.

Reply:
I don't think we published a version of this package *without* the *-dna*, https://www.npmjs.com/package/@microsoft/eslint-config-fast-dna. I think this should stay as-is.